wallet/invoices.h

```diff
@@ -213,8 +224,8 @@ void invoices_waitone(const tal_t *ctx,
  * @invoice - the invoice to get details on.
  * @return pointer to the invoice details allocated off of `ctx`.
  */
-const struct invoice_details *invoices_get_details(const tal_t *ctx,
-						   struct invoices *invoices,
-						   struct invoice invoice);
+struct invoice_details *invoices_get_details(const tal_t *ctx,
+					     struct invoices *invoices,
+					     struct invoice invoice);
```

Comment:
```suggestion
					     const struct invoices *invoices,
					     const struct invoice invoice);
```

Reply:
The 'struct invoice' is passed by value, so we don't generally const those.  And the invoices is not usually passed const to these functions, though it would be legal: we do change an invoice.

Reply:
ops, just noted now that invoice is a copy and we just don't care.